kotlin-styled-next/build.gradle.kts

```diff
@@ -33,22 +35,29 @@ dependencies {
 }
 
 val printBenchmarkResults by tasks.registering {
-    doLast {
-        // implicit dependency to benchmark classes names
-        val fileNames = listOf(
-            "AddStyledElements",
-            "ConvertToStyled",
-            "CssBuildersInject",
-            "AddDuplicateCss",
-            "DataTypeOperations",
-            "InjectCssNPlusOne"
-        )
-        fileNames.forEach { test ->
-            val report = layout.buildDirectory
-                .file("reports/tests/test/classes/benchmark.$test.html")
-                .get().asFile.readText()
+    dependsOn(tasks.allTests)
+
+    // implicit dependency to benchmark classes names
+    val fileNames = listOf(
+        "AddDuplicateCss",
+        "AddStyledElements",
+        "ConvertToStyled",
+        "CssBuildersInject",
+        "DataTypeOperations",
+        "InjectCssNPlusOne",
+    )
+    val reports = objects.fileCollection().from(fileNames.map { test ->
+        layout.buildDirectory.file("reports/tests/test/classes/benchmark.$test.html")
```

Comment:
I'm not sure this directory is correct. When I run tests, I can see two, but they're `jsTest` and `jsBrowserTest`

* `kotlin-styled-next/build/reports/tests/jsBrowserTest/`
* `kotlin-styled-next/build/reports/tests/jsTest/`

Do you know if it should really be `reports/tests/test/`?

Reply:
Probably folder changed after migration on `multiplatform` plugin.

cc @Leonya